converter/src/test/java/gov/cms/qpp/acceptance/PcfRoundTripTest.java

```diff
@@ -0,0 +1,62 @@
+package gov.cms.qpp.acceptance;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import gov.cms.qpp.acceptance.helper.JsonPathToXpathHelper;
+import gov.cms.qpp.conversion.Context;
+import gov.cms.qpp.conversion.decode.ClinicalDocumentDecoder;
+import gov.cms.qpp.conversion.encode.JsonWrapper;
+import gov.cms.qpp.conversion.model.validation.ApmEntityIds;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class PcfRoundTripTest {
```

Comment:
Reading the code, looks similar to what you were saying during sprint planning. Could you give a quick one-line comment here on the class? Don't think it's super necessary for the methods.

Reply:
Went ahead and added comments for both negative and positive test cases.